e2e/portalicious/pages/ProjectMonitoringPage.ts

```diff
@@ -0,0 +1,76 @@
+import { expect } from '@playwright/test';
+import { Locator, Page } from 'playwright';
+
+import BasePage from './BasePage';
+
+class ProjectMonitoring extends BasePage {
+  page: Page;
+  readonly peopleRegisteredTile: Locator;
+  readonly peopleIncludedTile: Locator;
+  readonly remainingBudgetTile: Locator;
+  readonly cashDisbursedTile: Locator;
+  readonly projectDescriptionTile: Locator;
+  readonly metricTileComponent: Locator;
+
+  constructor(page: Page) {
+    super(page);
+    this.page = page;
+    this.peopleRegisteredTile = this.page.getByTestId(
+      'metric-people-registered',
+    );
+    this.peopleIncludedTile = this.page.getByTestId('metric-people-included');
+    this.remainingBudgetTile = this.page.getByTestId('metric-remaining-budget');
+    this.cashDisbursedTile = this.page.getByTestId('metric-cash-disbursed');
+    this.projectDescriptionTile = this.page.getByTestId(
+      'metric-project-description',
+    );
+    this.metricTileComponent = this.page.getByTestId('metric-tile-component');
+  }
+
+  async assertMonitoringTabElements() {
+    const registrationsTileLocator = this.peopleRegisteredTile;
+    const includedTileLocator = this.peopleIncludedTile;
+    const remainingBudgetTileLocator = this.remainingBudgetTile;
+    const cashDisbursedTileLocator = this.cashDisbursedTile;
+    const projectDescriptionTileLocator = this.projectDescriptionTile;
+
+    const registrationTileText = await registrationsTileLocator.innerText();
+    const includedTileText = await includedTileLocator.innerText();
+    const remainingBudgetTileText =
+      await remainingBudgetTileLocator.innerText();
+    const cashDisbursedTileText = await cashDisbursedTileLocator.innerText();
+    const projectDescriptionTileText =
+      await projectDescriptionTileLocator.innerText();
+
+    expect(registrationTileText).toContain('People registered');
+    expect(includedTileText).toContain('People included');
+    expect(remainingBudgetTileText).toContain('Remaining budget');
+    expect(cashDisbursedTileText).toContain('Cash disbursed');
+    expect(projectDescriptionTileText).toContain('Project description');
+
+    // For the moment we are not checking the iframe area because there are no visible elemnts in it
```

Comment:
We could check "if its there or not", right?

Because for some programs/projects the URL would be configured, so then the `<iframe>` should be rendered.
And otherwise, if the property is missing, the "no iframe here, contact us"-text should be used.

But it could be a test-scenario for another day... ;)

Reply:
Yes the only issue I had was the iframe really I can take a look again into this. All the other comments should be resolved. In this case I changed "toHeaveText" into "toIncludeText" because it just checks for for one string.

Reply:
@Piotrk39 I was about to dismiss Elwin's review but I actually think it would be nice to add this.

I assume it would have to be something as simple as (pseudocode) `expect(page).toContainElement('iframe')` which should be true for one program and false for another.

Reply:
Moved to https://dev.azure.com/redcrossnl/121%20Platform/_workitems/edit/30569